client/packages/common/src/intl/locales/en/dispensary.json

```diff
@@ -8,6 +8,7 @@
   "button.new-patient": "New Patient",
   "button.new-prescription": "New Prescription",
   "button.unlink-patient": "Unlink Patient",
+  "button.view-stock-transaction": "View stock transaction",
```

Comment:
I know this is from the designs, but would "View prescription" be more appropriate?

Reply:
Probably yeah!